rust/connlib/tunnel/src/device_channel/tun_darwin.rs

```diff
@@ -19,7 +19,7 @@ use tokio::io::unix::AsyncFd;
 const CTL_NAME: &[u8] = b"com.apple.net.utun_control";
 
 #[derive(Debug)]
-pub(crate) struct Tun {
+pub struct Tun {
```

Comment:
This is pointless but it keeps us from having to write `target_os` directives in `device_channel.rs` to make it `pub` on Windows specifically.